connect/src/main/java/com/stripe/android/connect/webview/ChooseFileActivityResultContract.kt

```diff
@@ -0,0 +1,21 @@
+package com.stripe.android.connect.webview
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.webkit.WebChromeClient
+import androidx.activity.result.contract.ActivityResultContract
+
+/**
+ * Contract for launching a file chooser activity and handling its result.
+ * Used to select files for upload in Connect web flows.
+ */
+internal class ChooseFileActivityResultContract : ActivityResultContract<Intent, Array<Uri>?>() {
```

Comment:
Since the result depends on WebChromeClient's file chooser, should we rename this `WebChooseFileActivityResultContract` or similar?
```suggestion
internal class WebChooseFileActivityResultContract : ActivityResultContract<Intent, Array<Uri>?>() {
```